Exception: "<pending_xref py:class=\"AdalError\" py:module=\"adal.adal_error\" refdoc=\"\
  adal\" refdomain=\"py\" refexplicit=\"False\" reftarget=\"Exception\" reftype=\"\
  class\" refwarn=\"False\">\n    <literal classes=\"xref py py-class\">\n       \
  \ Exception\n"
dict: "<pending_xref py:class=\"AuthenticationContext\" py:module=\"adal.authentication_context\"\
  \ refdoc=\"adal\" refdomain=\"py\" refexplicit=\"False\" refspecific=\"True\" reftarget=\"\
  dict\" reftype=\"class\">\n    <literal_emphasis>\n        dict\n"
object: "<pending_xref py:class=\"TokenCacheKey\" py:module=\"adal.token_cache\" refdoc=\"\
  adal\" refdomain=\"py\" refexplicit=\"False\" reftarget=\"object\" reftype=\"class\"\
  \ refwarn=\"False\">\n    <literal classes=\"xref py py-class\">\n        object\n"
str: "<pending_xref py:class=\"AuthenticationContext\" py:module=\"adal.authentication_context\"\
  \ refdoc=\"adal\" refdomain=\"py\" refexplicit=\"False\" refspecific=\"True\" reftarget=\"\
  str\" reftype=\"class\">\n    <literal_emphasis>\n        str\n"
